src/multi-level-checkbox-editor/node-box.tsx

```diff
@@ -11,6 +11,7 @@ interface Props {
     onNodeClick: (node: Node) => void;
     setNodeValue: (node: Node, value: boolean) => void;
     levelHeaderLabel: string;
+    checkBoxHeader?: boolean;
```

Comment:
What about a rename of `checkBoxHeader` to `disableHeaderCheckbox`?

Reply:
yeah looks nice will change it